components/table/Table.razor.cs

```diff
@@ -238,6 +238,13 @@ public IEnumerable<TItem> DataSource
         [Parameter]
         public bool Bordered { get; set; } = false;
 
+        /// <summary>
+        /// Striped table or not
+        /// </summary>
+        /// <default value="false" />
+        [Parameter]
```

Comment:
Please add a attribute `[PublicApi("1.1.0")]`, then I will merge this PR. Thank you!